Cargo.toml

```diff
@@ -1,8 +1,16 @@
 [package]
+edition = "2021"
 name = "color_quant"
 license = "MIT"
 version = "1.1.0"
 authors = ["nwin <nwin@users.noreply.github.com>"]
 readme = "README.md"
 description = "Color quantization library to reduce n colors to 256 colors."
 repository = "https://github.com/image-rs/color_quant.git"
+
+[dependencies]
+# libm: "Uncomment if you wish to use `Float` and `Real` without `std`"
```

Comment:
This is a good place for information to users of the library. `Float` and `Real` are not traits of this library, so this comment seems to be misplaced. Or it should elaborate further what situation it applies to.